pdcupdater/handlers/modules.py

```diff
@@ -59,6 +63,42 @@ def can_handle(self, pdc, msg):
 
         return True
 
+    def get_unreleased_variant_rpms(self, pdc, variant):
```

Comment:
This method could use a docblock that describes its behavior and the purpose and types of its arguments.